lib/src/report/template.dart

```diff
@@ -261,3 +267,42 @@ Future<ReportSection> followsTemplate(PackageContext context) async {
     maxIssues: 10,
   );
 }
+
+Subsection _licenseSection(LicenseTags tags) {
+  const title = 'Uses OSI-approved license';
+  late List<Paragraph> paragraphs;
+  final maxPoints = 10;
+  var grantedPoints = 0;
+  var reportStatus = ReportStatus.failed;
+
+  String licensePluralized(List value) =>
+      value.length > 1 ? 'licenses' : 'license';
+  String licenseList(List<License> values) =>
+      values.map((e) => '`${e.spdxIdentifier}`').join(', ');
+  final detected = RawParagraph(
+      'Detected ${licensePluralized(tags.licenses)}: ${licenseList(tags.licenses)}.');
+
+  if (tags.isOnlyOsiApproved) {
+    paragraphs = [detected];
+    grantedPoints = maxPoints;
+    reportStatus = ReportStatus.passed;
+  } else if (tags.wasDetected) {
+    paragraphs = [
+      detected,
+      Issue(
+        'The following ${licensePluralized(tags.nonOsiApprovedLicenses)} are not OSI-approved: ${licenseList(tags.nonOsiApprovedLicenses)}.',
+        suggestion: 'Use only OSI-approved license in the `LICENSE` file.',
```

Comment:
This suggestion somehow comes out a bit harsh - the licensing of a package is the authors. Not sure how to make it better, maybe:
```suggestion
        suggestion: 'Consider using an OSI-approved license in the `LICENSE` file to share with the community.',
```